qualtran/bloqs/gf_arithmetic/gf2_multiplication.py

```diff
@@ -653,3 +660,333 @@ def _binarypolynomialmultiplication() -> BinaryPolynomialMultiplication:
 _BINARY_POLYNOMIAL_MULTIPLICATION_DOC = BloqDocSpec(
     bloq_cls=BinaryPolynomialMultiplication, examples=(_binarypolynomialmultiplication,)
 )
+
+
+@attrs.frozen
+class GF2ShiftRightMod(Bloq):
+    r"""Multiplies by $2^k$ (or $x^k$ for polynomials) modulu.
```

Comment:
Please replace `modulu` with `modulo` here and elsewhere. Also the sentence seems incomplete, `modulo` what ?

Reply:
done